test/fixtures/input/schema-registry/schema-ruleset.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domainRules": [
+    {
+      "name": "checkSsnLen",
+      "kind": "CONDITION",
+      "type": "CEL",
```

Comment:
Out of curiosity, what does "CEL" mean?

Reply:
"CEL" stands for Common Expression Language, it's an open-source project by Google.